src/components/crud-form-pages/OpaqueIngressSecret.js

```diff
@@ -18,17 +33,26 @@ export const OpaqueIngressSecret = (props) => {
       onSave={props.onSave}
       onSubmit={props.onSubmit}
       propsMatchState={props.propsMatchState}
+      hideFormTitleButton={props.noSecretsManager}
+      overrideTile={props.noSecretsManager ? <NoSecretsManagerTile /> : null}
       innerFormProps={{
+        noSecretsManager: props.noSecretsManager,
         secretsManagerList: props.secretsManagerList,
         secretsManagerGroupCallback: props.secretsManagerGroupCallback,
         secretsManagerGroupCallbackText: props.secretsManagerGroupCallbackText,
         secretCallback: props.secretCallback,
         secretCallbackText: props.secretCallbackText,
+        arbSecretCallback: props.arbSecretCallback,
```

Comment:
why are these different callbacks? can't you just use one for all secrets?

Reply:
Since there are 3 secrets, when they had the same callback before, if one secret name was invalid, they all became invalid. I added these so that they are treated separately.

Reply:
i think you need to change the logic in your callback to use different properties of the same `stateData` object